subgraphs/inventory/app/src/main/resources/graphql/inventory.graphqls

```diff
@@ -1,4 +1,12 @@
-extend schema @link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@key", "@external", "@requires", "@shareable"])
+extend schema
+    @link(url: "https://specs.apollo.dev/federation/v2.1",
```

Comment:
@korinne @pcmanus @clenfest @dariuszkuc: note that I had to bump the inventory subgraph to `@link` Fed v2.1 to successfully `import` the `@composeDirective` in `federation-jvm` in this inventory subgraph, but `subgraph-js` in the products subgraph below allows `@composeDirective` using `@link` with Fed v2.0.

Should we do additional validation in `subgraph-js` to consistently require Fed v2.1 for new directives introduced in that version of the Fed subgraph spec?